client/model_test.go

```diff
@@ -0,0 +1,21 @@
+package client_test
+
+import (
+	"encoding/json"
+
+	. "github.com/env0/terraform-provider-env0/client"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Models", func() {
+	Describe("TemplateCreatePayload", func() {
+		It("Should omit Github Installation Id when it's not there", func() {
```

Comment:
what about specify the id and check it exists in the payload

Reply:
when you're right you're right
https://github.com/env0/terraform-provider-env0/pull/63/commits/4f8e41a00788f47f1ee22b5197e17666b8806019